test/test_heuristics.rb

```diff
@@ -486,6 +486,14 @@ def test_tsx_by_heuristics
     })
   end
 
+  def test_v_by_heuristics
+    assert_heuristics({
+      "Coq" => all_fixtures("Coq", "*.v"),
+      "V" => ["V/loop.v"],
```

Comment:
Why test only `loop.v` for V?

Reply:
The heuristic didn't match most of the V samples (the ones used to train the classifier), so I deliberately targeted a file which contained the patterns being matched by V's heuristic.

Reply:
The other V files are correctly identified by the Bayesian classifier, rather than the Heuristic strategy (which isn't supposed to catch every fathomable V file).

Reply:
> The heuristic didn't match most of the V samples (the ones used to train the classifier)

I'm a bit surprised by that. I tested each sample file against the V heuristic and they all match at least the `^[ \t]*fn\s+[^\s()]+\(.*?\).*?\{` portion. What did I miss?

Reply:
> What did I miss?

Nothing, I'm just an idiot. :) 👍 I forgot I fixed an error in V's regex *after* seeing the failed tests, but I reran them again with `all_fixtures("V", "*.v")` and now it's passing, aaaaaand I'm just gonna stop talking and pretend like this never happened.

<img src="https://vignette.wikia.nocookie.net/men-in-black/images/e/eb/Neuralizer1.png">